pkg/cmd/openshift-controller-manager/cmd.go

```diff
@@ -162,5 +165,11 @@ func (o *OpenShiftControllerManager) RunControllerManager() error {
 		return err
 	}
 
-	return RunOpenShiftControllerManager(config, clientConfig)
+	if err := RunOpenShiftControllerManager(config, clientConfig, stopCh); err != nil {
+		return err
+	}
+
+	<-stopCh
```

Comment:
RunOpenShiftControllerManager blocks on stopCh already.

Reply:
yeah, figured, fixed.

Reply:
why so complicated? do it the other way around. Unneccessary concurrency is not nice.